specification/electronic-prescription-service-api.yaml

```diff
@@ -63,33 +56,48 @@ info:
+    ## Using this API as a dispenser
+    ### To download a prescription to be dispensed: 
     1. POST a [Task-release-message](https://simplifier.net/guide/DigitalMedicines/release-duplicate-2) to the `/FHIR/R4/Task/$release` endpoint to mark the prescription as released
     2. POST a [prescription-dispense-notification](https://simplifier.net/guide/DigitalMedicines/prescription-dispense) message to the `/FHIR/R4/$process-message` endpoint to inform SPINE of a partial/successful dispense
 
-    ## Legal Use
-    This API can only be used where there is a legal basis to do so. Make sure you have a valid use case before you go too far with your development.
-
-    You must do this before you can go live (see ‘Onboarding’ below).
+    ### To notify Spine that a prescription has been dispensed:
+    1. TBC
+    
+    ### To withdraw a prescription notification:
+    1. TBC
+    
+    ### To return a prescription to Spine:
+    1. TBC
 
     ## Related APIs
     The following APIs also give access to the Electronic Prescription Service:
     - [Electronic Prescription Service (HL7 V3) API](https://data.developer.nhs.uk/dms/mim/4.2.00/Index.htm) - use this if you want to use functions that are not yet available on the FHIR API
     - [Signing Service](https://digital.nhs.uk/developer/api-catalogue/signing-service) - use this to digitally sign a prescription
     - [Prescription Tracker API](https://developer.nhs.uk/apis/eps-tracker) - use this if you want a read-only interface to obtain information about a patient’s prescriptions
 
-    ## API status and roadmap
-    Prescribing flows for EPS FHIR API are in technical go-live. Dispensing flows for EPS FHIR API are in sandbox environment only.
-
-    The API is currently available for sandbox testing and integration testing, but not for production use.
+    The following APIs may be required when accessing the  Electronic Prescription Service:
+    - [Personal Demographics Service - FHIR API](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir) - use this to obtain patient data
+    
+    ## Additional guidance
+    For further details on common HTTP status codes, see [5xx status codes](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide)
 
+    ## API status and roadmap
     To see our roadmap, or to suggest, comment or vote on features for this API, see our [interactive product backlog](https://nhs-digital-api-management.featureupvote.com/?tag=eps-fhir-api).
 
     If you have any other queries, please [contact us](https://digital.nhs.uk/developer/help-and-support).
+    
+    ### Prescribing API
+    Prescribing flows for EPS FHIR API are now in a state of technical go-live awaiting our first of type integration.
+
+    The API is currently available for sandbox testing and integration testing, but not for production use.
+
+    ### Dispensing API
+    Dispensing flows for EPS FHIR API are in the sandbox environment only.
```

Comment:
only some parts of dispensing are in sandbox - withdraw and return are spec only